sdk/personalizer/Azure.AI.Personalizer/api/Azure.AI.Personalizer.netstandard2.0.cs

```diff
@@ -289,9 +362,9 @@ public partial class PersonalizerRankOptions
     public partial class PersonalizerRankResult
     {
         internal PersonalizerRankResult() { }
-        public string EventId { get { throw null; } }
-        public System.Collections.Generic.IReadOnlyList<Azure.AI.Personalizer.PersonalizerRankedAction> Ranking { get { throw null; } }
-        public string RewardActionId { get { throw null; } }
+        public string EventId { get { throw null; } set { } }
```

Comment:
Should these be settable via ctors and stay read-only, or does it make sense to keep them modifiable after the object is created?

Reply:
Removed the setter